views/layout.hbs

```diff
@@ -92,10 +41,10 @@
           <div class="content">
             <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
             <nav id="proposition-menu">
-              <a href="#availability" id="proposition-name">IRC Bed Management</a>
+              <a href="{{{ homepageUrl }}}" id="proposition-name">{{ title }}</a>
               <ul id="proposition-links" class="">
-                <li><a href="#availability" class="active">Bed availability</a></li>
-                <li><a href="#statistics">Stats</a></li>
+                <li><a href="{{ availabilityPath }}" class="active">{{ availabilityLink }}</a></li>
```

Comment:
Is there a reason that you've made the text into a variable?
I don't think it needs to be does it?